### PR TITLE
Python 3 builds

### PR DESCRIPTION
**Pull request by ocefpaf**
Fingers crossed!

### CODE REVIEW

**Comment by ocefpaf** [comment]
@rsignell-usgs AppVeyor is failing while downloading stuff from PyPI.  I am tempted to merge this as is and fix it as we go.

File: appveyor.yml [hunk 1]
```diff
@@ -8,15 +8,16 @@ environment:
...
 
   BINSTAR_TOKEN:
+    # Generated with "binstar auth --create --name comtbot -o ioos" and encrypted on ci.appveyor.com/tools/encrypt.
     secure: a0u2Qr8yHKQQHhFB+oWUg2tWeQYyUxHtFVk1ofwIPDaW9L5FrIC8fSns+Zbq0Kno
 
   matrix:
     - TARGET_ARCH: "x86"
       CONDA_PY: "27"
-      CONDA_NPY: "19"
+      PY_CONDITION: "\"python >2.7,<3\""
```
Comment (pelson) [inline appveyor.yml]: Do you know something I don't? I think you need ``>=2.7`` :wink:

File: cf_units/meta.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ build:
 
 requirements:
     build:
-        - python
+        - python <3
```
Comment (pelson) [inline cf_units/meta.yaml]: Apparently this is problematic. It means that conda doesn't bake the python version :cry: - I think I need to fix that in conda-build...

File: appveyor.yml (hunk 1 above)
Comment (ocefpaf) [inline appveyor.yml]: Oops :flushed:

File: cf_units/meta.yaml (hunk 2 above)
Comment (ocefpaf) [inline cf_units/meta.yaml]: To be safe we have to specify `<3` in build and run then?

Comment (pelson) [inline cf_units/meta.yaml]: Yes. And any future work to allow py3 support would need to be a new build number.

Comment (ocefpaf) [inline cf_units/meta.yaml]: Thanks!

**Pull request merged**